include/sys/param.h

```diff
@@ -43,6 +43,12 @@
 #  define MAX(a,b)      (((a) > (b)) ? (a) : (b))
 #endif  /* MAX */
 
+/* Macros for number of items. */
+
+#ifndef nitems
+#  define nitems(_a)    (sizeof((_a)) / sizeof((_a)[0]))
```

Comment:
```suggestion
#  define nitems(_a)    (sizeof(_a) / sizeof(0[(_a)]))
```
just to avoid possible `[]` overload in C++

Reply:
Done